docs/docs/how-to/dbauth-passwordless.md

```diff
@@ -0,0 +1,644 @@
+# Setting up dbAuth to be passwordless
+
+Security is really important.  Sometimes you don't want to integrate with a third-party authentication services.  Whatever the reason, Redwood has you covered with Redwood's dbAuth to authenticate users.  This is a great option.
+
+One thing though is now you're collecting the user's login, password and salt.  If you'd like to not collect all of this, an alternative is passwordless.  The only data needed for passwordless is the users email address.
+
+In this how-to I'll show you how to set up dbAuth to be passwordless, you'll still need to set up a a way to send [send emails](../how-to/sending-emails.md), but there's plenty of ways to do that.
```

Comment:
I reworded the intro.  I also added a link to the Send Email's how-to as that will have to be implemented to make this really work.

Reply:
Looks like there's a double "send" in the last sentence, one before the link and one inside the link.